### DISCUSSION TITLE
What does a highlighted message like this mean?

### DISCUSSION THREAD

**Discussion by GL513** [Chatterino/chatterino2#3780] [Q&A] [answered]
The user had more messages after that message that were also highlighted, but some also weren't. 
![GL513_22-05-29_16:12:56](https://user-images.githubusercontent.com/83085846/170889802-61e10587-2cb9-42cb-8f2f-3e6b5e9f0e57.png)
There was one other highlighted message after this one by this user, and then two un-highlighted ones, and then another _highlighted_ message soon after.

**Comment by LosFarmosCTL** [comment, marked as answer]
The highlight in the picture indicates that this was the first message this user has ever sent in that channel.

Whatever highlights came after that were something else.
